frontend/src/Pages/LoginPage.js

```diff
@@ -38,7 +38,7 @@ export default function LoginPage() {
   }
 
   return (
-    <Page>
+    <PageHeader>
       <div>
         <br />
         <img src={logo} alt="logo" />
```

Comment:
```suggestion
        <img src={logo} alt="logo" />
```

looks a bit hacky, try to solve layout problems in css not markup

Reply:
👍